src/manifest/parsers/testParser.ts

```diff
@@ -47,7 +47,10 @@ export class TestParser {
               database,
               schema,
               alias,
-              column_name,
+              // for quoted column names, remove the quotes
+              // ex: in manifest, it will be stored as "column_name": "\"Customer ID\"" for tests
+              // here we remove the extra quotes
+              column_name: column_name?.replace(/^"(.*)"$/, "$1"),
```

Comment:
Instead of using regexes can we not just use replace operations and replace any \" with " and remove the enclosing quotes?

Reply:
modified